theia-rpm-build-docker/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "theia",
+  "version": "0.8.0",
+  "license": "EPL-2.0",
+  "engines" : {
+	  "node":"10.x"
+  },
+  "dependencies": {
+      "@theia/callhierarchy": "latest",
+      "@theia/file-search": "latest",
+      "@theia/git": "latest",
+      "@theia/json": "latest",
+      "@theia/markers": "latest",
+      "@theia/merge-conflicts": "latest",
+      "@theia/messages": "latest",
+      "@theia/mini-browser": "latest",
+      "@theia/navigator": "latest",
+      "@theia/outline-view": "latest",
+      "@theia/preferences": "latest",
+      "@theia/preview": "latest",
+      "@theia/search-in-workspace": "latest",
+      "@theia/terminal": "latest",
+      "@theia/textmate-grammars": "latest",
+      "typescript": "latest"
+  },
+  "devDependencies": {
+      "@theia/cli": "latest"
+  },
+  "scripts": {
+      "prebuild": "npm install -g yarn",
+      "prebuild-rpm": "ln -s \"$( pwd )\" ~/rpmbuild && npm install -g speculate",
+      "build-rpm": "speculate && rpmbuild -bb SPECS/theia.spec",
+      "start": "npm run prebuild && yarn theia start"
+  },
+  "spec": {
```

Comment:
I am curious: I do not remember seeing a "spec" section in package.json before. Does the content end-up being used in the packaged Theia application?

Reply:
I think I got it: it's used by `build-rpm`?

Reply:
The `spec` section is used by the `speculate` module. See [here](https://rpm-packaging-guide.github.io/#rpm-packaging-workspace) for what a sample RPM dev workspace looks like. 

The `speculate` module basically generates the entire RPM workspace and all necessary metadata files (including the SPEC file) based on the settings defined in the `spec` section of `package.json`. Once these files are generated, calling the `rpmbuild` utility handles the rest of the packaging process.